CourseManager/Courses/Course 4 UI and Animations in SwiftUI/Views/Main/HomeViewWeather.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  HomeViewWeather.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-11-09.
+//
+
+import SwiftUI
+import BottomSheet
+
+enum BottomSheetPosition: CGFloat, CaseIterable {
+    case top = 0.83
```

Comment:
Good that you use enum, and better to add comments why this values are used

Reply:
This enum was used because a library for a pull up bottom sheet was included in the project. It uses this enum to know the position of the bottom sheet. You can define your own relative values or a default value is set. The instructor added her own values based on a figma file. I've updated the code as: 

enum BottomSheetPosition: CGFloat, CaseIterable {
    case top = 0.83 // 702/844 = position of sheet in 'top' position / overall height of screen
    case middle = 0.385 //325/844 = position of sheet in 'middle' position / overall height of screen
}

Reply:
@sghourab Just the point about such values. In general, you always want system values

Reply:
I'll keep that in mind for sure